templates/new-feature-email.html

```diff
@@ -1,56 +1,70 @@
-<section id="context">
-  <div>
-    <b>New</b> feature entry:<br/>
-    <a class="feature-name" href="{{SITE_URL}}feature/{{id}}"
-          >{{feature.name}}</a>
-  </div>
+{% import 'email-styles.html' as styles %}
+<div style="{{styles.body}}">
+<div style="{{styles.branding}}">{{APP_TITLE}}</div>
+
+<div style="{{styles.content}} {{styles.content_new}}">
+<section id="context" style="{{styles.section}}">
+  <table>
+    <tr>
+      <td rowspan=2>{{styles.icon_new()}}</td>
+      <td><b>New</b> feature entry:</td>
+    </tr>
+    <tr>
+      <td><a style="{{styles.feature_name}}"
+             href="{{SITE_URL}}feature/{{id}}"
+             >{{feature.name}}</a>
+      </td>
+    </tr>
+  </table>
 </section>
-<br/>
 
 
-<section id="details">
+<section id="details" style="{{styles.section}}">
   <div><b>New details entered by {{feature.creator_email}}:</div>
 
   <table>
     <tr>
-      <td>Summary:</td>
-      <td>{{feature.summary}}</td>
+      <td style="{{styles.td}}">Summary:</td>
+      <td style="{{styles.td}}">{{feature.summary}}</td>
     </tr>
     <tr>
-      <td>Unlisted:</td>
-      <td>{% if feature.unlisted %}Yes{% else %}No{% endif %}</td>
+      <td style="{{styles.td}}">Unlisted:</td>
+      <td style="{{styles.td}}">{% if feature.unlisted %}Yes{% else %}No{% endif %}</td>
     </tr>
     <tr>
-      <td>Breaking change:</td>
-      <td>{% if feature.breaking_change %}Yes{% else %}No{% endif %}</td>
+      <td style="{{styles.td}}">Breaking change:</td>
+      <td style="{{styles.td}}">{% if feature.breaking_change %}Yes{% else %}No{% endif %}</td>
     </tr>
     <tr>
-      <td>Feature owners:</td>
-      <td>{{feature.owner_emails|join(", ")}}</td>
+      <td style="{{styles.td}}">Feature owners:</td>
+      <td style="{{styles.td}}">{{feature.owner_emails|join(", ")}}</td>
     </tr>
     <tr>
-      <td>Cc:</td>
-      <td>{{feature.cc_emails|join(", ")}}</td>
+      <td style="{{styles.td}}">Cc:</td>
+      <td style="{{styles.td}}">{{feature.cc_emails|join(", ")}}</td>
     </tr>
     <tr>
-      <td>Blink component:</td>
-      <td>{{feature.blink_components|join(", ")}}</td>
+      <td style="{{styles.td}}">Blink component:</td>
+      <td style="{{styles.td}}">{{feature.blink_components|join(", ")}}</td>
```

Comment:
I notice that the resulting text looks like, e.g. Blink>Compositing>Filters, which should probably have space around the ">" chars.   

Also, I am wondering what join(", ") is joining in that case.  Can there be multiple blink components, each one being a path through the hierarchy?

Reply:
In the UI, there are no spaces around the `>` characters, so I'll leave that as-is for consistency.

Yes, the component is a repeated string field in our NDB models, but our UI only exposes the option to set a single value for now.